3p/integration.js

```diff
@@ -56,6 +56,7 @@ import {criteo} from '../ads/criteo';
 import {dotandads} from '../ads/dotandads';
 import {doubleclick} from '../ads/google/doubleclick';
 import {eplanning} from '../ads/eplanning';
+import {fakead3p} from '../ads/fakead3p';
```

Comment:
The import is fine, it'll get DCE'd by closure. We just can't use the imported reference if we're in prod.